data/distro.yml

```diff
@@ -586,3 +586,11 @@
     - name: Restart
       text: '
         <p>To complete setup, restart your system. Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>'
+
+- name: Vanilla OS
+  logo: "vanillaos.svg"
+  introduction: >
+    <h2>Flatpak is installed by default on Vanilla OS.</h2>
+    <p>You can use the Software app or browse <a href="https://flathub.org/">Flathub</a> to install some flatpak apps.</p>
+    <p>If for some reason Flathub is not available, you can configure it manually using the following command:</p>
+    <terminal-command>host-shell flatpak remote-add --user --if-not-exists flathub https://dl.flathub.org/repo/flathub.flatpakrepo</terminal-command>
```

Comment:
Why the `--user` switch?

Reply:
According to [the documentation](https://docs.vanillaos.org/handbook/en/install-flatpaks) Vanilla os only uses/officially supports flatpak with the user switch

Reply:
At Vanilla OS we don't support system flatpak's as the size of the underlying host system is small (and also to avoid `sudo`, since our host image doesn't have it by design, we substitute it with Polkit policies).

Reply:
This line has also been requested in https://github.com/flathub-infra/website/pull/3700#issuecomment-2277802728

Reply:
Ah, thanks for the clarification!